util/test/sub_test

```diff
@@ -240,11 +240,30 @@ def PerfDirFile(s):
 def PerfTFile(test_filename, sfx):
     return test_filename + '.' + perflabel + sfx
 
-# read file with comments
+# Read a file or if the file is executable read its output. If the file is
+# executable, the current chplenv is copied into the env before executing.
+# Expands shell variables and strip out comments/whitespace. Returns a list of
+# string, one per line in the file.
 def ReadFileWithComments(f, ignoreLeadingSpace=True):
```

Comment:
I think it'd be out of place since we don't use docstrings anywhere else in sub_test